changelog/@unreleased/pr-6232.v2.yml

```
@@ -0,0 +1,9 @@
+type: fix
+fix:
+  description: |-
+    See PDS-293157.
+    Presently, on each host, the `lastSweptTimestamp` metric is updated lazily per shard as sweep progresses. If a particular host, for some reason, is sweeping with a low frequency, its `lastSweptTimestamp` metric will be out of sync with `shardProgress` and its `millisSinceLastSwept` will show sweep falling behind.
+
+    This PR makes it so we update that metric periodically for all shards by reading and updating from one source: the persisted `shardProgress` table.
```

```suggestion
    The `lastSweptTimestamp` metric for targeted sweep is now updated at regular intervals by reading persisted sweep progress. Previously, this was done lazily, which could result in stale metrics during periods with few writes.
```
We don't generally mention internal stuff in open source repositories, which includes PDS tickets. Aim to make these messages more concise and to the point